### PR TITLE
Fix build errors and add missing docs

### PR DESCRIPTION
**Pull request by Krakaw**

### CODE REVIEW

File: yat-docs/docs/partner-guides/1_crypto_wallets.md [hunk 1]
```diff
@@ -272,6 +272,27 @@ So for example, to query the BTC address only you can simply call
...
+    "video": "https://y.at/viz/poison/poison.octopus.poison-f4fc44.mp4",
+    "webm": "https://y.at/viz/poison/poison.octopus.poison-f4fc44.webm"
+  },
+  "is_locked": false,
+  "created_at": "2021-08-03T08:51:00.362249Z",
+  "updated_at": "2021-09-15T09:23:37.787489Z",
+  "locked_future_writes_at": null
+}
+```
+
```
Comment (djangolin) [inline yat-docs/docs/partner-guides/1_crypto_wallets.md:295]:
```suggestion

Coming soon: If **no visualization** exists for the Yat, we recommend partners display the fallback graphic in place of the visualization. Fallback graphics will be updated on to this documentation shortly. 
```

File: yat-docs/docs/partner-guides/1_crypto_wallets.md [hunk 2]
```diff
@@ -294,5 +315,24 @@ As a part of this flow, the partner app will need to:
 * Assign a nickname to the address
 * Allow users to enter/paste the Yat instead of their wallet address when sending money
 
+## 8. Invalid Yats
+
+Looking up a valid Yat, that has not yet been registered will return a 404 error
```
Comment (djangolin) [inline yat-docs/docs/partner-guides/1_crypto_wallets.md:320]:
```suggestion
Looking up a valid Yat, that has not yet been registered will return a 404 error.
```

File: yat-docs/docs/partner-guides/1_crypto_wallets.md [hunk 3]
```diff
@@ -272,6 +272,27 @@ So for example, to query the BTC address only you can simply call
...
+    "webm": "https://y.at/viz/poison/poison.octopus.poison-f4fc44.webm"
+  },
+  "is_locked": false,
+  "created_at": "2021-08-03T08:51:00.362249Z",
+  "updated_at": "2021-09-15T09:23:37.787489Z",
+  "locked_future_writes_at": null
+}
+```
+
 ## 5. Receive payments from a Yat
```
Comment (Krakaw) [inline yat-docs/docs/partner-guides/1_crypto_wallets.md:297]:
```suggestion

Looking up a Yat from an address is not currently possible, however the client application could store a map of used addresses to known Yats and confirm the address is still linked to that Yat when doing the lookup.
```

Comment (djangolin) [reply yat-docs/docs/partner-guides/1_crypto_wallets.md:298]: Add link to Address Book flow

File: yat-docs/docs/partner-guides/1_crypto_wallets.md [hunk 4]
```diff
@@ -272,8 +272,33 @@ So for example, to query the BTC address only you can simply call
...
+  "updated_at": "2021-09-15T09:23:37.787489Z",
+  "locked_future_writes_at": null
+}
+```
+
+Coming soon: If **no visualization** exists for the Yat, we recommend partners display the fallback graphic in place of the visualization. Fallback graphics will be updated on to this documentation shortly.
+
 ## 5. Receive payments from a Yat
 
+Looking up a Yat from an address is not currently possible, however the client application could store a map of used addresses to known Yats and confirm the address is still linked to that Yat when doing the lookup. Using Flow [7](#7-add-yats-to-address-book)
```
Comment (djangolin) [inline yat-docs/docs/partner-guides/1_crypto_wallets.md:300]:
```suggestion
Looking up a Yat from an address is not currently possible, however the client application could store a map of used addresses to known Yats and confirm the address is still linked to that Yat when doing the lookup by using Flow [7](#7-add-yats-to-address-book). 
```

**Review by djangolin — APPROVED** [review]

**Pull request merged**